pkg/tests/cross-tests/diff_cross_test.go

```diff
@@ -51,39 +51,191 @@ func TestUnchangedBasicObject(t *testing.T) {
 	})
 }
 
-func TestSimpleStringNoChange(t *testing.T) {
-	config := map[string]any{"name": "A"}
-	runDiffCheck(t, diffTestCase{
-		Resource: &schema.Resource{
-			Schema: map[string]*schema.Schema{
-				"name": {
-					Type:     schema.TypeString,
-					Optional: true,
-				},
+func TestDiffBasicTypes(t *testing.T) {
+	res := &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"other_prop": {
+				Type:     schema.TypeString,
+				Optional: true,
 			},
 		},
-		Config1: config,
-		Config2: config,
-	})
-}
+	}
 
-func TestSimpleStringRename(t *testing.T) {
-	runDiffCheck(t, diffTestCase{
-		Resource: &schema.Resource{
-			Schema: map[string]*schema.Schema{
-				"name": {
-					Type:     schema.TypeString,
-					Optional: true,
+	typeCases := []struct {
```

Comment:
It's not testing ForceNew support it seems.

Reply:
ForceNew is tested in the replace test cases for each type: https://github.com/pulumi/pulumi-terraform-bridge/blob/aafaaf854308ae0a486ebff1ea410a56985868de/pkg/tests/cross-tests/diff_cross_test.go#L220